src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(Color.values().length);
+        return Color.values()[index].toString();
```

Comment:
The `getRandomColor` method returns a `String`, but it would be more appropriate to return a `Color` type. This ensures type safety and consistency when using the method. Consider changing the return type to `Color` and returning `Color.values()[index]` directly.